packages/server/__tests__/setup-tests.js

```diff
@@ -0,0 +1,13 @@
+const server = require("../index");
+const { closePostgres } = require("../utils/postgres");
+
+/**
+ * utility function to clean up and close database connection and express
+ * server after testing
+ */
+function cleanUpConnections() {
+  closePostgres();
+  server.close();
+}
+
+module.exports = cleanUpConnections;
```

Comment:
I am slightly confused on why this file is necessary. If there were multiple functions needed to initialize the tests to use Jest, it would be fine. Since only one function is created here, its indiviudal function calls can be made in `index.test.js` within the `afterAll`.

Reply:
As you said, I wrote this utility file anticipating any new setup/teardown associated with possible dependencies we'll add to the project in the future. It'll be easier to do all of this in one function to be imported and used in Jest `beforeAll()` and `afterAll()` hooks. I only created `index.test.js` as a template for new team members to reference when creating new tests in the future like `events.test.js` and `users.test.js` for the events and users controllers.